src/NuGet.Core/NuGet.Packaging/Signing/NuGetChainBuildingRequestType.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NuGet.Packaging.Signing
+{
+    enum NuGetChainBuildingRequestType
```

Comment:
specify an access modifier, either public or internal. Per the coding guidelines for nuget we avoid leaving it off